site/packages/aa-accounts/light-account/client.md

```diff
@@ -35,21 +35,21 @@ A Promise containing a new `SmartAccountClient` connected to a Light Account.
 
 - `transport: Transport` -- a Viem transport for interacting with JSON RPC methods.
 
-- `chain: Chain` -- the chain on which to create the provider.
+- `chain: Chain` -- the chain on which to create the client.
 
 - `owner: SmartAccountSigner` -- the owner EOA address responsible for signing user operations on behalf of the smart account.
 
-- `entryPointAddress: Address | undefined` -- [optional] the entry point contract address. If not provided, the entry point contract address for the provider is the connected account's entry point contract, or if not connected, falls back to the default entry point contract for the chain. See [getDefaultEntryPointAddress](/packages/aa-core/utils/getDefaultEntryPointAddress.html#getdefaultentrypointaddress).
+- `entryPointAddress: Address | undefined` -- [optional] the entry point contract address. If not provided, the entry point contract address for the client is the connected account's entry point contract, or if not connected, falls back to the default entry point contract for the chain. See [getDefaultEntryPointAddress](/packages/aa-core/utils/getDefaultEntryPointAddress.html#getdefaultentrypointaddress).
```

Comment:
I think I forgot to update these docs, but this doesn't take an address any more I don't think. It takes in an EntryPointDef

Reply:
this still takes in an Address. this is the optional override entry point address that one can pass in to override the default

Reply:
I think we should remove this here because `ToSmartContractAccountParams` already takes in entrypointdef and has the address field. will remove.